backend/src/test/java/com/github/chuettenrauch/mixifyapi/integration/mixtape/controller/MixtapeControllerTest.java

```diff
@@ -178,31 +183,42 @@ void getAll_whenLoggedIn_thenReturnMixtapesForTheLoggedUser() throws Exception {
     }
 
     @Test
-    void delete_whenLoggedInButMixtapeDoesNotExist_thenReturnNotFound() throws Exception {
+    void delete_whenLoggedInButCanNotEdit_thenReturnForbidden() throws Exception {
         OAuth2User oAuth2User = this.testUserHelper.createLoginUser();
```

Comment:
Ich hätte hier vll den Namen des Tests mit ...ButCanNotEdit*BecauseIsOnlyListener* ergänzt, ich hab den Nutzen erst verstanden, als ich auf den nächsten Test geguckt hab

Reply:
Warte, ich bin hier bisschen verwirrt, was für ein Szenario von diesem Test abgedeckt wird

Reply:
Hab die test beschreibungen mal geupdated. 
Das is der fall, wenn das einfach ein mixtape von nem komplett anderen user is und man damit gar nix zu tun hat